src/services/bip39-service/bip39-service.spec.ts

```diff
@@ -182,6 +184,56 @@ describe("BIP39Service", () => {
             }
         );
     });
+
+    it("should return a string array from a string and when already retrieved return it locally", (done) => {
+        spyOn(httpClient, "get").and.returnValue(Observable.of("word1, word2, word3"));
```

Comment:
The data passed in the promise is not correctly formatted. Instead it should be written as:
```
Observable.of(`
word1
word2
word3
`)
```

This is also the reason the data you are checking on is `['word1, word2, word3']` and not `['word1', 'word2', 'word3']` which it should be.